integration-tests/src/test/java/com/datastax/oss/driver/api/querybuilder/JacksonJsonCodec.java

```diff
@@ -98,27 +98,16 @@ public String format(T value) {
   public T parse(String value) {
     if (value == null || value.isEmpty() || value.equalsIgnoreCase("NULL")) return null;
```

Comment:
In driver 4 the contribution guidelines mandate that if/for/while blocks must be enclosed with braces. Sam ething for the line below.